applications/nrf_desktop/configuration/nrf54h20dk_nrf54h20_cpuapp/app.overlay

```diff
@@ -4,6 +4,11 @@
  * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
  */
 
+/* For nRF54H, watchdog status is disabled by default. Needs to be enabled in DTS overlay. */
```

Comment:
I wonder if we shouldn't introduce `app_release.overlay` for that purpose - watchdog is used only by the release configuration (similar note for nRF54L). We used to keep it in common DTS overlay, because some time ago build system used not to support DTS overlays per build type.

Reply:
Could you create a task for it?

Reply:
Created: `NCSDK-30399`